client/components/suggestions/index.jsx

```diff
@@ -32,7 +32,8 @@ const Suggestions = React.createClass( {
 	getInitialState() {
 		return {
 			taxonomySuggestionsArray: [],
-			suggestionPosition: -1,
+			suggestionPosition: 0,
+			currentSuggestion: undefined,
```

Comment:
I've considered that, but I think it's generally a good idea to explicitly initiate component state...